### DISCUSSION TITLE
Sync data between devices

### DISCUSSION THREAD

**Discussion by SorrisoKDente** [kodjodevf/mangayomi#230] [Ideas]
It would be great to have a built-in sync function for Mangayomi, similar to what Syncyomi  has for Tachiyomi through Google Drive. This would be incredibly helpful for users who access their library across multiple devices. It would automatically keep your library, history, and configuration data up-to-date, eliminating the need for manual backups every time you want to switch devices.

**Comment by fenglengshun** [comment]
I saw that there is a sync option in Mangayomi now? How do I connect my PC (well, Steam Deck) and phone?

**Reply by SorrisoKDente** [reply]
I tried this and worked
https://github.com/kodjodevf/mangayomi/issues/382#issuecomment-2668635779

**Reply by fenglengshun** [reply]
Whose server is that? I'd rather just use my own device to sync and self-host.

**Reply by SorrisoKDente** [reply]
If you want to use your own server, you can follow the steps described in this repository:
https://github.com/mangayomi/mangayomi-sync

**Reply by Whiskas101** [reply]
Isn't this a pure metadata sync? What about having a central home server and just accessing stuff over the network?

**Reply by shirooo39** [reply]
> Isn't this a pure metadata sync? What about having a central home server and just accessing stuff over the network?

If you want to have a central library and read from anywhere, maybe you want to try [suwayomi-server](https://github.com/Suwayomi/Suwayomi-Server) with the [sorayomi](https://github.com/Suwayomi/Tachidesk-Sorayomi) app
